bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -89,7 +89,7 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
   concord::util::SimpleThreadPool internalThreadPool;  // TODO(GG): !!!! rename
 
   // retransmissions manager (can be disabled)
-  RetransmissionsManager* retransmissionsManager = nullptr;
+  unique_ptr<RetransmissionsManager> retransmissionsManager = nullptr;
```

Comment:
nit: you can remove the nullptr.
The Initial state of a smart pointer is always null.

Reply:
Done